cmd/multiFaSequenceSwap/multiFaBedSequenceSwap.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/bed"
+	"github.com/vertgenlab/gonomics/fasta"
+	"github.com/vertgenlab/gonomics/fileio"
+	"github.com/vertgenlab/gonomics/numbers/parse"
+	"log"
+)
+
+// SubsequenceSwap swaps a series of specified regions from a bed file between two sequences in a MultiFa object
+// The sequence called "background" is taken, and parts of "foreground" are swapped with the corresponding parts in background
```

Comment:
Let's clarify the language a bit. The key detail to emphasis is that the bed regions are the foreground, and everything outside the beds is the background